File: Java/0001.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long a = scanner.nextLong();
        long b = scanner.nextLong();
        long sum = a + b;
        System.out.println(sum);
        scanner.close();
    }
}

File: Kotlin/0001.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val input = scanner.nextLine()
    val numbers = input.split(" ")

    val var1 = numbers[0].toInt()
    val var2 = numbers[1].toInt()

    val sum = var1 + var2
    println(sum)
}